openshift/system/config/backend.yml

```diff
@@ -1,13 +1,7 @@
-base: &default
-  host: backend-listener:3000
-  public_url: <%= ENV.fetch("BACKEND_ROUTE") { "https://backend-#{ENV['TENANT_NAME']}.#{ENV['THREESCALE_SUPERDOMAIN']}" } %>
-
 production:
-  <<: *default
-  secure: false
-  persistent: true
-
-preview:
-  <<: *default
-  secure: false
-  persistent: true
+  url: <%= ENV.fetch('BACKEND_URL', 'http://backend-listener:3000') %>
```

Comment:
`url` here is to replace `host` + `secure`, see also https://github.com/3scale/porta/pull/3107/files#r1024235636

the operator does not currently set `BACKEND_URL`, so the behavior should not change.

`public_url` changes only the quotes `"BACKEND_ROUTE"` to `"BACKEND_ROUTE'`